include/net/cloud.h

```diff
@@ -112,19 +112,15 @@ struct cloud_event {
 	} data;
 };
 
-/**
- * @brief Cloud last will and testament type. See:
- * http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/os/mqtt-v3.1.1-os.html
- * sections  3.1.2.5-3.1.2.7, 3.1.3.2-3.1.3.3.
- */
-struct cloud_lwt {
-	u8_t qos;		/**< MQTT QoS value. */
-	/* Broker to retain will for new clients.  Not currently
-	 * supported by AWS IoT.
-	 */
-	bool retain;
-	const char *topic;	/**< If specified, @ref message must also. */
-	const char *message;	/**< Message sent to topic by broker. */
+/**@brief Cloud configuration. */
+struct cloud_cfg {
+	union {
+		struct {
+			struct cloud_msg epitath_msg;
```

Comment:
When it's in an MQTT specific struct, I'd vote to call it lwt_msg or something to make it sound familier to people familiar with the concept